.circleci/config.yml

```diff
@@ -87,6 +89,26 @@ jobs:
       - run-bazel-rbe:
           command: bazel test //common/... --test_output=errors
 
+  test-server:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run-bazel-rbe:
+          command: bazel test //server/... --test_output=errors
+
+  test-graql:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run-bazel-rbe:
+          command: bazel test //graql/... --test_output=errors
```

Comment:
I thought @flyingsilverfin had added this in a previous PR? Why does it need to be added again?